routes/api/admin.js

```diff
@@ -0,0 +1,68 @@
+const router = require('express').Router();
```

Comment:
Why do we have a route called admin?

Reply:
I have renamed it to role.js, I initially used a name that can easily be remembered